record-two/record.controller.js

```diff
@@ -0,0 +1,10 @@
+(function() {
+    'use strict';
+
+    angular.module('chaise.record')
+
+    .controller('RecordController', ['$rootScope', function FormController($rootScope) {
```

Comment:
The function name doesn't _really_ matter but `FormController` should be `RecordController` for consistency.